npm/src/s3upload/index.ts

```diff
@@ -16,69 +16,76 @@ type TdrS3 = Pick<S3, "upload">
 
 export class S3Upload {
   s3: TdrS3
-  identityId: string
 
-  constructor(identityId: string, region: string) {
+  constructor() {
     const timeout = 20 * 60 * 1000
     const connectTimeout = 20 * 60 * 1000
-    this.s3 = new S3({ region, httpOptions: { timeout, connectTimeout } })
-    this.identityId = identityId
+    this.s3 = new S3({ httpOptions: { timeout, connectTimeout } })
```

Comment:
How come the `region` is no longer needed?

Reply:
I think it's because we were sending the requests to https://tdr-upload-files-dirty-intg.s3.eu-west-2.amazonaws.com so it needed the region to construct the url but now we're overriding it, it doesn't bother so it doesn't need it.

Reply:
Fair enough.